### ISSUE TITLE
Django complains about SQLite native datetime

### ISSUE THREAD

**Issue by matematikaadit** [matematikaadit/matematikaadit.github.io#30]
# TLDR

In your `setting.py` file, add following content:

``` python
# ignore the following error when using ipython
# django/db/backends/sqlite3/base.py:58: RuntimeWarning:
# SQLite received a naive datetime ...
# link to SO: http://stackoverflow.com/a/13194250/1072844

import warnings
import exceptions
warnings.filterwarnings("ignore",
    category=exceptions.RuntimeWarning,
    module='django.db.backends.sqlite3.base',
    lineno=58) # change lineno to your line number warning
```
# Issue

This workaround was found in this SO answer: http://stackoverflow.com/a/13194250/1072844

Version of affected program:
- Django 1.6

**What happened**

When starting `python manage.py shell` in IPython, it gives following warning:

```
/home/adit/.virtualenv/local/lib/python2.7/site-packages/
django/db/backends/sqlite3/base.py:58: RuntimeWarning: SQLite received a naive 
datetime (2013-12-10 22:36:49.148282) while time zone support is active.
  RuntimeWarning)
```

Take a note of the file that give the warning (django/db/backends/sqlite3/base.py)
the line number (58) and the exception type (RuntimeWarning). Your case might be
different.

On your `settings.py` add following content:

```
import warnings
import exceptions
warnings.filterwarnings("ignore",
    category=exceptions.RuntimeWarning,
    module='django.db.backends.sqlite3.base',
    lineno=58) # change lineno to your line number warning
```

change `category`, `module`, and `lineno` according what you note previously.
